.travis.yml

```diff
@@ -13,6 +13,7 @@ env:
 
 install:
   - npm install
+  - sudo pip install -r requirements.txt
```

Comment:
I have tried but it is not working but when I'm having nodejs as language and `sudo pip install` the build is passing